tests/test_job_status_retrieval_strategy.py

```diff
@@ -127,23 +127,24 @@ def test_no_profile_stderr_file(self, tmp_path: Path) -> None:
         result = self.js.get_job_status(tmp_path)
         assert not result.is_successful
         assert result.error_message == (
-            f"profile_stderr.txt file not found in the specified output directory, {tmp_path}. "
-            "This file is expected to be created during the profiling stage. "
+            f"No profile_stderr_*.txt files found in the specified output directory, {tmp_path}. "
+            "These files are expected to be created during the profiling stage. "
             "Please ensure the profiling stage completed successfully. "
             "Run the generated sbatch script manually to debug."
         )
 
     def test_missing_pax_status_keyword(self, tmp_path: Path) -> None:
-        """Test that job status is False when profile_stderr.txt does not contain the PAX STATUS keyword."""
-        profile_stderr_file = tmp_path / "profile_stderr.txt"
+        """Test that job status is False when profile_stderr_*.txt does not contain the PAX STATUS keyword."""
+        profile_stderr_file = tmp_path / "profile_stderr_1.txt"
```

Comment:
What is the reason for renaming profile_stderr.txt to profile_stderr_1.txt?

Reply:
@TaekyungHeo Let me know if that makes sense.